### PR TITLE
Fix typos in Field-Caps documentation

### PR DESCRIPTION
**Pull request by dmeiss**

### CODE REVIEW

File: docs/reference/search/field-caps.asciidoc [hunk 1]
```diff
@@ -3,15 +3,15 @@
...
+The field capabilities API by default executes on all indices:
 
 [source,js]
 --------------------------------------------------
 GET _field_caps?fields=rating
 --------------------------------------------------
 // CONSOLE
 
-... but the request can also be restricted to specific indices:
+...but the request can also be restricted to specific indices:
```
Comment (cbuescher) [inline docs/reference/search/field-caps.asciidoc]: How about omitting the dots altogether, starting a new sentence with "The requestst can...". I think that would be much better.

**Review by cbuescher — CHANGES_REQUESTED** [review]
Hi @dmeiss, thanks for opening this PR to add changes to our docs. I left a small suggestion for improvement, would you be able to add those changes to the PR?

File: docs/reference/search/field-caps.asciidoc (hunk 1 above)
Comment (dmeiss) [reply docs/reference/search/field-caps.asciidoc]: I agree -- good suggestion. I'll make the change.

**Review by cbuescher — APPROVED** [review]
Great, thanks @dmeiss for the additional changes. Will merge this after running some tests.

**Comment by cbuescher** [comment]
@elasticmachine test this please

**Pull request merged**